checkstyle/import-control-core.xml

```diff
@@ -142,4 +142,9 @@
     <allow pkg="org.apache.directory" />
     <allow pkg="org.apache.mina.core.service" />
   </subpackage>
+
+  <subpackage name="clients">
+    <allow pkg="org.apache.kafka.clients" />
+    <allow pkg="org.apache.log4j" />
+  </subpackage>
```

Comment:
Do we really need this?

Reply:
Yes, without this, we will get `Disallowed import` in `./gradlew checkstyleMain checkstyleTest spotlessCheck`.

Reply:
Is `<allow pkg="org.apache.log4j" />` still required with the last changes?

Reply:
Sorry, I missed it. Yes, we don't need it.